src/components/Notes/UpdateNoteDialog.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+
+const UpdateNoteDialog = ({ open, onClose, note, onSave }) => {
```

Comment:
You should start boolean props with `is`

```suggestion
const UpdateNoteDialog = ({ isOpen, onClose, note, onSave }) => {
```